metadata.rb

```diff
@@ -1,14 +1,14 @@
-# coding: UTF-8
-
 maintainer 'Cerner Corp'
 maintainer_email 'Bryan.Baugher@Cerner.com'
-license 'All rights reserved'
+license 'Apache-2.0'
 description 'Installs/Configures tomcat'
 long_description 'This cookbook is meant to install and configure an instance of tomcat'
 issues_url 'https://github.com/cerner/cerner_tomcat/issues'
 source_url 'https://github.com/cerner/cerner_tomcat'
 name 'cerner_tomcat'
 
+chef_version '>= 12.0' if respond_to?(:chef_version)
```

Comment:
What's the purpose of the respond_to? Why can't we just specify ">= 12.0"? And actually, given the problems we've encountered recently with Chef 13 should we just have "~ 12.0" for now?